classes/Hook.php

```diff
@@ -1244,4 +1250,30 @@ private static function getMethodName(string $hookName): string
     {
         return 'hook' . ucfirst($hookName);
     }
+
+    /**
+     * Return an instance of the specified hook.
+     *
+     * @param string $hook_name Hook name
+     *
+     * @return bool
+     */
+    public static function getStatusByName($hook_name): bool
+    {
+        if (empty($hook_name)) {
+            return true;
+        }
+
+        $hook_datas = Db::getInstance()->getRow('
+			SELECT `active`
+			FROM `' . _DB_PREFIX_ . 'hook`
+			WHERE `name` = "' . pSQL($hook_name) . '"
+		');
+
+        if (empty($hook_datas)) {
+            return true;
```

Comment:
Should be false, the hook is not found, it should not be considered as active :thinking:

Reply:
Some hooks are not present in the database. You can't toggle their status, so I'm assuming they're active by default. For example https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Dispatcher.php#L596

Reply:
If hooks are not present in the database, the status is false by default.
It's like not going to school because of disease, you're not active in the classroom, you can't do anything.